harvester/sources/extraction/edurep.py

```diff
@@ -0,0 +1,286 @@
+import re
+from mimetypes import guess_type
+from hashlib import sha1
+from core.constants import HIGHER_EDUCATION_LEVELS
+from django.conf import settings
+from dateutil.parser import parse as date_parser
+from datagrowth.processors import ExtractProcessor
+from django.utils.text import slugify
+
+
+FILE_TYPE_TO_MIME_TYPE = {
```

Comment:
This is problematic. We need to talk about this with Edurep if we really can't find the mime_type. Do they want us to look for ourselves?! This value is much more akin to our technical_type than it is to a mime_type.

Reply:
For the record ... that file was HKU and HKU's "API" is a bunch of JSON files on a server somewhere with inconsistent typing. We shouldn't lower the bar that far if possible.